### PR TITLE
Spelling: Various language fixes

### PR DESCRIPTION
**Pull request by comradekingu**
I'm not 100% sure the "we" strings weren't in error.

### CODE REVIEW

**Comment by nerzhul** [comment]
sounds good except the welcome message, can you restore default message ?

thanks

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -181,10 +181,10 @@
     </string-array>
 
     <!-- Main activity -->
-    <string name="ma_title_rate_us">Rate us!</string>
+    <string name="ma_title_rate_us">Rate us.</string>
     <string name="ma_title_add_account">Add an account</string>
     <string name="ma_title_welcome">Welcome</string>
-    <string name="ma_content_welcome">Welcome to Nextcloud SMS application. This application let you synchronize your SMS to your Nextcloud instance using SMS app.</string>
```
Comment (comradekingu) [inline src/main/res/values/strings.xml]: Says Welcome twice (?) (The string preceeding it is "Welcome".)
Needs "the" Nextcloud SMS application.
Says "Application" twice.
Should be "lets".
Need to qualify "SMS" with messages.
Using "an"(?) SMS app.

**Comment by comradekingu** [comment]
@nerzhul It has quite a few errors in there, could you look at it. Feel free to edit.

File: src/main/res/values/strings.xml (hunk 1 above)
Comment (nerzhul) [inline src/main/res/values/strings.xml]: nice job

**Pull request merged**